src/System.Diagnostics.Process/src/System.Diagnostics.Process.builds

```diff
@@ -11,6 +11,9 @@
     <Project Include="System.Diagnostics.Process.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Diagnostics.Process.csproj">
+      <TargetGroup>net461</TargetGroup>
+    </Project>
     <Project Include="facade\System.Diagnostics.Process.csproj">
```

Comment:
@weshaggard  : I did think about it, but just left it unchanged assuming the facade folder clean up would take care of it. I could unify them all and remove the facade folder( and have the assemblyVersion set to 4.0 or 4.1 based on a condition checking the targetGroup). Would you prefer that i do that?